### ISSUE TITLE
[CRASH] InvalidCharacterError : String contains an invalid character

### ISSUE THREAD

**Issue by sgtoutlaw** [EliteMasterEric/Teyvat.moe#301]
**Describe the crash**
Try to export a route I made for my personal use and I got this error

**Steps To Produce**

Try and export and this error shows up.

I cannot test whether it happens when there isn't any data in the edits, because I don't want to lose my current ones.

**Device Specs (please complete the following information):**

- App Version: 10.6
- Device: pc
- OS: Windows 10
- Browser: firefox
- Browser Version: 85

**Error Message**

```

InvalidCharacterError : String contains an invalid character
t@https://genshinmap.github.io/static/js/2.70cf55f4.chunk.js:2:4756437
E<@https://genshinmap.github.io/static/js/2.70cf55f4.chunk.js:2:4845044
z@https://genshinmap.github.io/static/js/2.70cf55f4.chunk.js:2:4831057
div
c<@https://genshinmap.github.io/static/js/2.70cf55f4.chunk.js:2:4782695
R<@https://genshinmap.github.io/static/js/2.70cf55f4.chunk.js:2:4832994
w<@https://genshinmap.github.io/static/js/2.70cf55f4.chunk.js:2:4847252
u/</C<@https://genshinmap.github.io/static/js/2.70cf55f4.chunk.js:2:243612
div
Ji@https://genshinmap.github.io/static/js/main.47603ad3.chunk.js:1:2778213
div
D/t</</f<@https://genshinmap.github.io/static/js/2.70cf55f4.chunk.js:2:4809909
div
D/t</</f<@https://genshinmap.github.io/static/js/2.70cf55f4.chunk.js:2:4809909
vo@https://genshinmap.github.io/static/js/main.47603ad3.chunk.js:1:2732860
div
D/t</</f<@https://genshinmap.github.io/static/js/2.70cf55f4.chunk.js:2:4809909
hi@https://genshinmap.github.io/static/js/main.47603ad3.chunk.js:1:2763044
Xi<@https://genshinmap.github.io/static/js/main.47603ad3.chunk.js:1:2781853
w@https://genshinmap.github.io/static/js/2.70cf55f4.chunk.js:2:247530
div
D/t</</f<@https://genshinmap.github.io/static/js/2.70cf55f4.chunk.js:2:4809909
vo@https://genshinmap.github.io/static/js/main.47603ad3.chunk.js:1:2732860
div
D/t</</f<@https://genshinmap.github.io/static/js/2.70cf55f4.chunk.js:2:4809909
ru<@https://genshinmap.github.io/static/js/main.47603ad3.chunk.js:1:2791098
w@https://genshinmap.github.io/static/js/2.70cf55f4.chunk.js:2:247530
div
w@https://genshinmap.github.io/static/js/2.70cf55f4.chunk.js:2:247530
f@https://genshinmap.github.io/static/js/2.70cf55f4.chunk.js:2:245428
e@https://genshinmap.github.io/static/js/main.47603ad3.chunk.js:1:2681790
t.a<@https://genshinmap.github.io/static/js/2.70cf55f4.chunk.js:2:4805770
u/</C<@https://genshinmap.github.io/static/js/2.70cf55f4.chunk.js:2:243612
t@https://genshinmap.github.io/static/js/2.70cf55f4.chunk.js:2:431044
t.a@https://genshinmap.github.io/static/js/2.70cf55f4.chunk.js:2:4804878

```

**Comment by EliteMasterEric** [comment]
Can you press `F12`, type `window.localStorage`, and copy paste what comes up?

Apologies for the MYRIAD issues with the map the past couple of days, whenever I try to push a fix for one thing, it seems another thing breaks. Or when the project successfully builds on my computer, it doesn't build on the server or vice versa.

**Comment by EliteMasterEric** [comment]
Route exporting appears to be fixed on v0.10.8.
